src/Maintenance/Maintenance.Test/Maintenance.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Maintenance" Version="1.6.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Maintenance" Version="1.7.0-preview" />
```

Comment:
there is no such package:
https://www.nuget.org/packages/Microsoft.Azure.Management.Maintenance
and we do not support track 2 SDK